web/src/App.scss

```diff
@@ -20,3 +20,7 @@
   height: 100vh;
   background-color: green;
 }
+
+.app-body {
+  padding-bottom: general.$navbar-height
+}
```

Comment:
Missing a newline here. This should be being fixed by vscodes editior config plugin if its installed.